pwvlt-cli/src/main.rs

```diff
@@ -112,9 +112,10 @@ fn handle_store_errors(err: PassStoreError) {
         ),
         PassStoreError::PasswordNotFound => error!("No password could be found!"),
         PassStoreError::SkipError => unimplemented!("SkipError"),
-        PassStoreError::LockedError => error!(
-            "An error occurred while unlocking the Nitrokey backend"
-        ),
+        PassStoreError::LockedError => {
+            error!("An error occurred while unlocking the Nitrokey backend")
+        }
+        PassStoreError::Utf8(_) => error!("Failed to parse a password as Utf8"),
```

Comment:
No, since the only  place which can throw such an error is 
```rust
            let secret_bytes = item.get_secret()?;
            Ok(String::from_utf8(secret_bytes)?)
```